app/views/hq/sponsors/_index.html.haml

```diff
@@ -0,0 +1,43 @@
+- if sponsors.empty?
```

Comment:
user#show has two sponsor indexes, that's why it waited on the osra330 merge.  This just reformats the existing view to be reusable by User